scipy/stats/morestats.py

```diff
@@ -3085,7 +3059,8 @@ def wilcoxon(x, y=None, zero_method="wilcox", correction=False,
     return WilcoxonResult(T, prob)
 
 
-def median_test(*args, **kwds):
+def median_test(*args, ties='below', correction=True, lambda_=1,
```

Comment:
The old default for lambda_ was `None`, did you intend to change to it to `1`?

Reply:
The `lambda_` parameter is simply passed on to `chi2_contingency`, where the value `None` is equivalent to using `1`.     In `median_test`, the default for `lambda_` is documented in the docstring as being `1`, so I made that the explicit default value.